lunes.go

```diff
@@ -80,6 +80,11 @@ var dayPeriodsStd = []string{
 	"PM",
 }
 
+var dayPeriodsStdLower = []string{
```

Comment:
nitpick: if we're calling this `dayPeriodsStdLower`, the other should be `dayPeriodsStdUpper`